packages/migrations/src/Component/Doctrine/Migrations/MigrationsConfig.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Shopsys\MigrationBundle\Component\Doctrine\Migrations;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class MigrationsConfig
+{
+    const MIGRATIONS_CONFIG_FILE_NAME = 'migrations_config.yml';
+
+    /**
+     * @var string
+     */
+    private $migrationsConfigPath;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->migrationsConfigPath = $container->getParameter('shopsys.root_dir') . '/' . self::MIGRATIONS_CONFIG_FILE_NAME;
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getMigrationsConfig()
```

Comment:
The naming isn't clear: method `MigrationsConfig::getMigrationsConfig()` returning array of strings is confusing...